mail_all/static/description/index.html

```diff
@@ -0,0 +1,32 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Show all messages</h2>
+            <h3 class="oe_slogan">Checkout all messages where you have access</h3>
+        </div>
+        <div class="oe_row oe_spaced">
+            <div class="oe_span12">
+                <p class="oe_mt32">
+                    The module adds usual menu. This menu shows all messages.
```

Comment:
The module adds usual menu, that shows all messages. It differs from Arhives, because it show messages where current User is not participate. Example for Administrator: